doc/rocks.txt

```diff
@@ -1,22 +1,57 @@
 ==============================================================================
-Table of Contents                                               *rocks.contents*
+rocks.nvim                                                          *rocks-nvim*
+
+
+A luarocks plugin manager for Neovim.
 
-rocks.nvim ························································ |rocks.nvim|
-rocks.nvim commands ··········································· |rocks.commands|
-rocks.nvim configuration ········································ |rocks.config|
-Lua API for rocks.nvim extensions ·································· |rocks.api|
-rocks.nvim API hooks ········································· |rocks.api.hooks|
-rocks.nvim logging API ············································· |rocks.log|
 
 ==============================================================================
-rocks.nvim                                                          *rocks.nvim*
+Table of Contents                                               *rocks-contents*
 
+rocks.nvim ························································ |rocks-nvim|
+ ·································································· |rocks-toml|
+rocks.nvim commands ··········································· |rocks-commands|
+rocks.nvim configuration ········································ |rocks-config|
+Lua API for rocks.nvim extensions ·································· |rocks-api|
+rocks.nvim API hooks ········································· |rocks-api-hooks|
+rocks.nvim logging API ············································· |rocks-log|
 
-A luarocks plugin manager for Neovim.
+==============================================================================
+                                                                    *rocks-toml*
```

Comment:
shouldn't the tag be `rocks.toml` ? or maybe rocks-rocks.toml to avoid conflicting tags ?

Reply:
ha the dots have been changed in the rest of the doc so I speculate it's wanted

Reply:
Yep, at least for me, `tagfunc` doesn't seem to jump to the tags if it's a dot.